agent/util-scripts/gold/pbench-clear-tools/test-68.txt

```diff
@@ -23,6 +24,6 @@ Removed "vmstat" from host, "fubar5.example.com", in tools group, "default"
 --interval: 42
 --- pbench tree state
 +++ pbench.log file contents
-[info][1900-01-01T00:00:00.000000] Removed "vmstat" from host, "fubar5.example.com", in tools group, "default"
-[debug][1900-01-01T00:00:00.000000] Tool "vmstat" not registered for remote "fubar6.example.com"
+[INFO][1900-01-01T00:00:00.000000] Removed "vmstat" from host, "fubar5.example.com", in tools group, "default"
+[DEBUG][1900-01-01T00:00:00.000000] Tool vmstat not registered for remote fubar6.example.com
```

Comment:
Can we add the quotes back here?  We have them in the `INFO` message above, but not in this `DEBUG` message.

Reply:
fixed